backend/game/newcards/discard_cards/victory_point_manipulation_cards/test_gas_refinery.py

```diff
@@ -0,0 +1,17 @@
+from game.newcards.discard_cards.victory_point_manipulation_cards.gas_refinery import \
+    GasRefinery
+
+
+def test_gas_refinery_costs_6_energy():
+    assert GasRefinery().cost == 6
+
+
+def test_gas_refinery_adds_2_victory_points(player, five_players):
```

Comment:
adds 2 victory points to whom?

Reply:
Modified test case name to "test_gas_refinery_adds_2_victory_points_to_current_player"